app/assets/javascripts/components/Newsletter.js

```diff
@@ -0,0 +1,60 @@
+define(['jquery', 'DoughBaseComponent'], function($, DoughBaseComponent) {
+
+  'use strict';
+
+  var Newsletter,
+      defaultConfig = {};
+
+  Newsletter = function($el, config) {
+    Newsletter.baseConstructor.call(this, $el, config, defaultConfig);
+  };
+
+  /**
+  * Inherit from base module, for shared methods and interface
+  */
+  DoughBaseComponent.extend(Newsletter);
+
+  /**
+  * @param {Promise} initialised
+  */
+  Newsletter.prototype.init = function(initialised) {
+    this._initialisedSuccess(initialised);
+    this.showInArticle();
+    return this;
+  };
+
+  Newsletter.prototype.showInArticle = function() {
+    var $panel = this.getPanel();
+    var $article = $('[data-article]');
+
+    this.moveToArticle($panel, $article);
+
+    $panel.addClass('news-signup-test--with-image news-signup-test--in-article');
+  };
+
+  Newsletter.prototype.moveToArticle = function($panel, $article) {
+    var article = $article[0],
+        placed = false,
+        minimumPosition = article.offsetHeight / 4;
```

Comment:
The `article` variable is only used once so maybe `$article[0]` would be better.